frontend/src/components/SelectColumn/SelectColumn.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { FunctionComponent } from 'react';
+import { useSortable } from '@dnd-kit/sortable';
+import { CSS } from '@dnd-kit/utilities';
+
+interface SelectColumnProps {
+  id: string;
+}
+
+const SelectColumn: FunctionComponent<SelectColumnProps> = (props) => {
+  const { attributes, listeners, setNodeRef, transform, transition } = useSortable({
+    id: props.id,
+  });
+  const style = {
+    borderStyle: 'solid',
+    width: '400px',
+    margin: '2px',
+    cursor: 'grab',
+    transform: CSS.Translate.toString(transform),
+    transition: transition?.toString(),
+  };
```

Comment:
Its called `styled-components`... even if it didn't exist, this is discouraged unless you're doing animations.

Reply:
Why is it discouraged?